site/src/components/CodeExample/CodeExample.tsx

```diff
@@ -74,6 +82,7 @@ export const CodeExample: FC<CodeExampleProps> = ({
       </code>
 
       <CopyButton ref={buttonRef} text={code} />
+      <EyeButton ref={buttonRef} toggleHide={toggleHide} toShow={toShow}/>
```

Comment:
Rather than having a dedicated `toShow` prop, it might be better to have conditional rendering in this component that ties the output to whether we should be showing the secret

Reply:
Also, I would need to test this, but my gut feeling is that attaching `buttonRef` to multiple child components at once could cause either infinite event loops or at least cause events to be dispatched more than necessary

The purpose of the ref logic was to make it so that the entire token element could be clickable – without breaking the underlying HTML semantics for screen readers